src/telemetry/index.js

```diff
@@ -8,10 +8,6 @@ import GleanMetrics from "@mozilla/glean/metrics";
 
 import { googleAnalytics } from "./ga";
 
-// Glean generated metrics and pings files
-import * as pageMetrics from "./generated/page";
```

Comment:
We still need to remove the metrics and pings from the YAML files, but leave the yaml files there (without metrics)